packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { MouseEvent, useMemo, useState } from 'react';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { useModal } from '../../../../../components/ModalProvider';
+import { StaticLink } from '../../../../../components/StaticLink';
+import DocumentsIcon from '../../../../../public/images/ic-documents.svg';
+import IdCardIcon from '../../../../../public/images/ic-id-card.svg';
+import SelfieIcon from '../../../../../public/images/ic-selfie.svg';
+import { useDynamicLeverageModalState } from '../../../components/DynamicLeverageContext';
+import { getJurisdictionContents } from '../jurisdiction-contents/jurisdiction-contents';
+import { TJurisdictionCardItems, TJurisdictionCardSection } from '../jurisdiction-contents/props.types';
+import {
+    JurisdictionCardClass,
+    JurisdictionCardClassProps,
+    JurisdictionCardTagClass,
+    JurisdictionCardTagProps,
+} from './JurisdictionCard.classnames';
+import JurisdictionCardBack from './JurisdictionCardBack';
+import JurisdictionCardRow from './JurisdictionCardRow';
+import JurisdictionCardTag from './JurisdictionCardTag';
+import JurisdictionCardVerificationTag from './JurisdictionCardVerificationTag';
+
+type TJurisdictionCardProps = JurisdictionCardClassProps & {
+    jurisdiction: string;
+    onSelect: (clickedJurisdiction: string) => void;
+    tag?: string;
+};
+
+type TDisplayTextSkinColor = JurisdictionCardTagProps['displayTextSkinColor'];
+
+type TVerificationDocumentsMapper = {
+    [key: string]: {
+        category: 'poa' | 'poi' | null;
+        icon?: JSX.Element;
+    };
+};
+
+const verificationDocumentsMapper: TVerificationDocumentsMapper = {
+    documentNumber: {
+        category: 'poi',
+        icon: <IdCardIcon />,
+    },
+    nameAndAddress: {
+        category: 'poa',
+        icon: <DocumentsIcon />,
+    },
+    notApplicable: {
+        category: null,
+    },
+    selfie: {
+        category: 'poi',
+        icon: <SelfieIcon />,
+    },
+};
+
+const JurisdictionCard = ({
+    isAdded = false,
+    isSelected = false,
+    jurisdiction = 'svg',
+    onSelect,
+}: TJurisdictionCardProps) => {
+    const [isFlipped, setIsFlipped] = useState(false);
+    const { toggleDynamicLeverage } = useDynamicLeverageModalState();
+    const { getModalState } = useModal();
+
+    const descriptionClickHandler = (tag?: TJurisdictionCardProps['tag']) => (event: MouseEvent) => {
+        event.stopPropagation();
+        if (tag === 'dynamicLeverage') {
+            toggleDynamicLeverage();
+        } else {
+            setIsFlipped(true);
+        }
+    };
+
+    const { contents, header, isOverHeaderAvailable, overHeader, verificationDocs } = useMemo<TJurisdictionCardItems>(
+        () => getJurisdictionContents()[jurisdiction],
+        [jurisdiction]
+    );
+    const marketType = getModalState('marketType') || 'all';
```

Comment:
can refer to MT5AccountTypeModal regarding this

Reply:
For now there is no Ui and MT5AccountTypeModal is using useState for MarketType.